controller/src/beerocks/master/db/db.h

```diff
@@ -508,7 +508,7 @@ class db {
     };
 
     struct bss_info_conf_t {
-        uint8_t operating_class;
+        std::list<uint8_t> operating_class;
```

Comment:
As usual, I'm trying to learn: why use std::list instead of std::vector (and vice versa)?